spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :system do
+  before do
+    driven_by(:rack_test)
+    @user = User.create(name: 'John', photo: 'https://kiddy.com/pic/890987655', bio: 'Hi there', posts_counter: 0)
+    @post = Post.create(title: 'Hello from Mars',
+                        text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.', comments_counter: 0, likes_counter: 0, author_id: @user.id) # rubocop:disable Layout/LineLength
+
+    visit user_post_path(@user, @post)
+  end
+
+  context 'When visiting the post show page' do
+    it "should display the post's title" do
+      expect(page).to have_content(@post.title)
+    end
+
+    it "should display the author's name" do
+      expect(page).to have_content('John')
+    end
+
+    it 'should display the number of comments' do
+      expect(page).to have_content('Comments:0')
+    end
+    it 'should display the number of likes' do
+      expect(page).to have_content('Likes: 0')
+    end
+    it "should display the post's body" do
+      expect(page).to have_content(@post.text)
+    end
+
+    it 'should display the comments pusblished' do
+      @post.comments.each do |comment|
+        expect(page).to have_content(comment.text)
+        expect(page).to have_content(comment.user.name)
+      end
+    end
```

Comment:
Hi there, the following test required test cases are not present in your post index spec file for testing please add them, make sure they pass and submit them for review again 😄 👍🏾

- Tests that asserts you can see the username of each commentor.